test/checks/aria/aria-prohibited-attr.js

```diff
@@ -64,4 +64,20 @@ describe('aria-prohibited-attr', function() {
     );
     assert.isFalse(checkEvaluate.apply(checkContext, params));
   });
+
+  it('should allow whitelisted nodes to have aria-label', function () {
```

Comment:
We should also avoid the term whitelist

```suggestion
  it('should allow passed nodes to have aria-label', function () {
```